src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -118,17 +118,34 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="InitialJsonParseErrorText" xml:space="preserve">
-    <value>Settings could not be loaded from file - temporarily using the default settings. Check for syntax errors, including trailing commas.</value>
+    <value>Settings could not be loaded from file - temporarily using the default settings. Check for syntax errors, including trailing commas.
+</value>
+  </data>
+  <data name="MissingDefaultProfileText" xml:space="preserve">
+    <value>Could not find your default profile in your list of profiles - temporarily using the first profile. Check to make sure the defaultProfile matches the guid of one of your profiles.

```

Comment:
Curious: what do you mean by "temporarily" here and below? Maybe remove it?

Are these warnings here because (1) WT may behave differently from what the user expects or (2) WT may suddenly crash when your doing something later?

Reply:
Yea I guess that "temporarily" doesn't seem useful for warnings. Kinda more confusing with it than without.